Naggum.Assembler/AssemblyInfo.fs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Naggum.Assembler.AssemblyInfo
+
+open System.Reflection
+open System.Runtime.InteropServices
+
+[<assembly: AssemblyTitle("Naggum.Assembler")>]
+[<assembly: AssemblyProduct("Naggum")>]
+[<assembly: AssemblyCopyright("Copyright © Codingteam 2015")>]
```

Comment:
Use proper copyrights in the new code. I'll create an issue to actualize that for the existing code.